tools/hlint.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
-
```

Comment:
Why remove this?

Reply:
From the PR description:

> I also disalbed the pipefail from the script, as that would short-circuit the linter on first issue found. Hopefully that doesn't mess with CI.

Does that make sense?

Reply:
Ah, sorry I didn't read the description! I am sure there is way we can find, but I don't have much time to spend on this, so I will take back my objection.